### PR TITLE
refactor package structure

### PR DESCRIPTION
**Pull request by svenwltr**
Refactoring packages, because it's hip, fancy and cool.

### CODE REVIEW

**Comment by svenwltr** [comment]
@rebuy-de/prp-kubernetes-deployment Please review.

File: cmd/app.go [hunk 1]
```diff
@@ -8,8 +8,8 @@ import (
 
 	log "github.com/Sirupsen/logrus"
 
-	"github.com/rebuy-de/kubernetes-deployment/kubernetes"
-	"github.com/rebuy-de/kubernetes-deployment/settings"
+	"github.com/rebuy-de/kubernetes-deployment/pkg/kubernetes"
+	"github.com/rebuy-de/kubernetes-deployment/pkg/settings"
```
Comment (das-vinculum) [inline cmd/app.go]: Why is everything moved into a `pkg` sub package?

Comment (svenwltr) [inline cmd/app.go]: Because in this way it is easier to distinguish between golang packages and other directories, like `hack` and `target`.

Comment (das-vinculum) [inline cmd/app.go]: Okay, fine

**Pull request merged**